recipes-devtools/rust/compiler-rt_1.12.1.bb

```diff
@@ -0,0 +1,60 @@
+SUMMARY = "Rust compiler run-time"
+HOMEPAGE = "http://www.rust-lang.org"
+SECTION = "devel"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://src/compiler-rt/LICENSE.TXT;md5=bf24bca27049b52e9738451aa55771d4"
+
+SRC_URI = "\
+	https://static.rust-lang.org/dist/rustc-${PV}-src.tar.gz;name=rust \
+    "
+
+require rust.inc
+require rust-source-${PV}.inc
+
+DEPENDS += "rust-llvm-native (=${PV})"
+
+S = "${WORKDIR}/rustc-${PV}"
+
+DISABLE_STATIC = ""
+INHIBIT_DEFAULT_RUST_DEPS = "1"
+
+do_configure () {
+    # rust.inc generates these into ${WORKDIR}/mk-cfg/ but
+    # save ourselves time and patching by moving them back
+    # to the normal spot
+    mv "${WORKDIR}"/mk-cfg/*.mk "${S}/mk/cfg/" || bbfatal "couldn't move platform cfg"
+
+    ${S}/configure                          \
+        "--enable-rpath"                    \
+        "--disable-docs"                    \
+        "--disable-manage-submodules"       \
+        "--disable-debug"                   \
+        "--enable-optimize"                 \
+        "--enable-optimize-cxx"             \
+        "--disable-llvm-version-check"      \
+        "--llvm-root=${llvmdir}"            \
+        "--enable-optimize-tests"           \
+        "--release-channel=stable"          \
+        "--prefix=${prefix}"                \
+        "--target=${TARGET_SYS}"            \
+        "--host=${HOST_SYS}"                \
+        "--build=${BUILD_SYS}"              \
+        "--localstatedir=${localstatedir}"  \
+        "--sysconfdir=${sysconfdir}"        \
+        "--datadir=${datadir}"              \
+        "--infodir=${infodir}"              \
+        "--mandir=${mandir}"                \
+        "--libdir=${libdir}"                \
+        "--enable-local-rust"               \
+        "--local-rust-root=${WORKDIR}/${RUST_SNAPSHOT}/rustc" \
+        ${EXTRA_OECONF}
+}
+
+do_compile () {
+	oe_runmake ${TARGET_SYS}/rt/libcompiler-rt.a
+}
+
+do_install () {
+	mkdir -p ${D}${libdir}
+	cp ${TARGET_SYS}/rt/libcompiler-rt.a ${D}${libdir}/libcompiler-rt.a
```

Comment:
You did at https://github.com/meta-rust/meta-rust/pull/51#issuecomment-243815407